profile_library/signify/LCX002/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-12-11T02:55:33.655121",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "shelly,S3PM-001PCEU16",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.16.2:docker"
+  },
+  "name": "Play Gradient Lightstrip 65 Inch",
+  "standby_power": -0.55
```

Comment:
I notice the standby power is negative?
Did the measure script put this in or manually?

Reply:
I noticed that too. The measure script was used.
I had a few questions:
1. Is the standby power measure at the end of the test? If so I wonder if the the resistance of my dummy load decreased to cause that.
2. Is it possible to get a negative power reading on the Shelly because of polarity?

Reply:
What did you use as dummy load exactly?

> I noticed that too. The measure script was used. I had a few questions:
> 
> 1. Is the standby power measure at the end of the test? If so I wonder if the the resistance of my dummy load decreased to cause that.

The standby power is measured at the end of the script. Light is turned off, the script waits 20 seconds, and then takes a power reading.

> 2. Is it possible to get a negative power reading on the Shelly because of polarity?

Unsure.

Could you try to measure standby_power manually again. You can also use the `average` option in the measure tool to measure the standby power during X amount of time.

Reply:
> What did you use as dummy load exactly?

I used a 25W incandesce bulb. I can use a different dummy load if there are other recommendations. My best guess as the bulb heated up over time and its resistance decreased and decreased the power.

Here is the average power when the bulb is first plugged in:
2024-12-13 16:39:42,948 [INFO] Average power: 25.59
This was over 60 seconds.
Here is an hour later:
2024-12-13 17:40:02,654 [INFO] Average power: 25.23

Reply:
@dewbot6 hmmm, yes that would probably be the case.
Could you try starting the average power measurement first only with the bulb connected?
Let it run for 1 minute and not the W value.
Next connect the Hue to it and let it run for another 1 minute.
Compare both average values, that should give us a pretty accurate standby value.

Reply:
@bramstroker here is with the just the bulb:
[INFO] Average power: 25.90
Here is with the light strip on but in standby:
[INFO] Average power: 25.90
Interestingly the standby power looks to be 0?

Reply:
Very strange power is not being updated.
0 cannot be correct. All smart lights consume a bit of power, ranging from 0.2 to 1W typically. Radio needs to be active to receive signals from your remote / bridge.
Hue light range from 0.2 - 0.5W typically.

According to specs of the manufacturer this light has a standby power consumption of 0.50W.
https://www.philips-hue.com/en-gb/p/hue-white-and-colour-ambiance-play-gradient-lightstrip-65-inch/8718699784775#specifications

So when you are not able to measure (unsure why) I would suggest to set it to 0.5W

Reply:
@bramstroker I measured again and same issue. Confirmed that the light is on as when I turn the brightness up power usage goes up. I updated the standby to 0.5W.